src/components/ButtonComponents/NumberButtons/NumberButton.js

```diff
@@ -1,9 +1,12 @@
 import React from "react";
 
-const NumberButton = () => {
+const NumberButton = props => {
```

Comment:
This is a valid way of passing in props, but destructuring them is a good habit to get into because it increases readability (e.g. props should be {numbersMapped}).